spec/json_column_spec.cr

```diff
@@ -0,0 +1,62 @@
+require "./spec_helper.cr"
+
+class BlobQuery < Blob::BaseQuery
+end
+
+class BlobForm < Blob::BaseForm
+  fillable :doc
+end
+
+describe "JSON Columns" do
+  it "should work in boxes" do
+    BlobBox.create
+    blob = BlobQuery.new.first
+    blob.doc.should eq JSON::Any.new({"foo" => JSON::Any.new("bar")})
```

Comment:
Would `({"foo" => "bar"}).to_json` work here as well? If so I think it would be more readable since calling `JSON::Any.new` is verbose. Also you can convert NamedTuples too like `({ foo: "bar" }).to_json`